Assets/Scripts/Ingame TileManager/placeTile.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Tilemaps;
+public class placeTile : MonoBehaviour {
+
+    /* TODO 
+    - change "testTile" to "tileSelected" 
+    - implement "SelectTile" method
+    
+    */
+
+    [SerializeField]
+    Tile testTile;
+
+    Tile tileSelected;
+    [SerializeField]
+    Tilemap tilemap;
+
+    Camera cam;
+
+    void Start () {
+        cam = Camera.main;
+    }
+
+    void Update () {
+        if (Input.GetMouseButtonDown (0)) {
```

Comment:
Curly braces go on their own line
```suggestion
        if (Input.GetMouseButtonDown (0))
        {
```